crates/rpc/rpc-eth-types/src/fee_history.rs

```diff
@@ -421,8 +401,11 @@ impl FeeHistoryEntry {
     ///
     /// Returns a `None` if no excess blob gas is set, no EIP-4844 support
     pub fn next_block_excess_blob_gas(&self) -> Option<u64> {
-        self.excess_blob_gas.and_then(|excess_blob_gas| {
-            Some(self.blob_params?.next_block_excess_blob_gas(excess_blob_gas, self.blob_gas_used?))
+        self.header.excess_blob_gas().and_then(|excess_blob_gas| {
+            Some(
+                self.blob_params?
+                    .next_block_excess_blob_gas(excess_blob_gas, self.header.blob_gas_used()?),
```

Comment:
keeping this operating on the header and not moving into chainspec for now makes sense